### PR TITLE
Added new astronauts - worth checking?

### PR DESCRIPTION
**Pull request by chrisnorth**
I updated the astronauts for latest missions. Probably worth double checking, but seems correct...

### CODE REVIEW

File: human-spaceflight/data/KOCH_CH.md [hunk 1]
```diff
@@ -15,11 +15,21 @@ country:
   - USA
 missions:
   - name: Soyuz MS-12
-    time_start:	2019-03-14T19:14Z
-    time_end:	
+    time_start: 2019-03-14T19:14Z
+    time_end:   2020-02-06T00:00Z
```
Comment (christiansandberg2045) [inline human-spaceflight/data/KOCH_CH.md]: should be: time_end: 2020-02-06T09:12Z

File: human-spaceflight/data/KOCH_CH.md [hunk 2]
```diff
@@ -15,11 +15,21 @@ country:
...
 evas:
   - time_start: 2019-03-29T11:42Z
     time_end:   2019-03-29T18:27Z
+  - time_start: 2019-10-06T11:39Z
+    time_end:   2019-10-06T18:40Z
+  - time_start: 2019-10-11T11:38Z
+    time_end:   2019-10-11T18:23Z
+  - time_start: 2019-10-11T11:38Z
+    time_end:   2019-10-18T18:55Z
+  - time_start: 2020-01-15T11:25Z
```
Comment (christiansandberg2045) [inline human-spaceflight/data/KOCH_CH.md]: Should be - time_start: 2020-01-15T11:35Z ( http://www.spacefacts.de/iss/english/exp_61.htm )

File: human-spaceflight/data/MEIR_JU.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+country:
+  - USA
+missions:
+  - name: Soyuz MS-15
+    time_start: 2019-09-25T13:57:43Z
+    time_end:
+evas:
+  - time_start: 2019-03-29T11:42Z
+    time_end:   2019-03-29T18:27Z
+  - time_start: 2020-01-15T11:25Z
```
Comment (christiansandberg2045) [inline human-spaceflight/data/MEIR_JU.md:27]: Should be - time_start: 2020-01-15T11:35Z

File: human-spaceflight/data/MORGAN_AR.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+birthplace:
+  - name:	Morgantown, West Virginia
+    latitude:	39.633611
+    longitude:	-79.950556
+qualifications:
+  - BSc in Environmental Engineering
+  - Medical Doctorate
+country:
+  - USA
+missions:
```
Comment (christiansandberg2045) [inline human-spaceflight/data/MORGAN_AR.md:15]: **Should be:** http://www.spacefacts.de/iss/english/exp_60.htm
missions:
  - name: Soyuz MS-13
    time_start: 2019-07-20T16:28Z
    time_end:
  - name: Soyuz MS-15
    time_start:
    time_end: 2020-04-17T05:16Z

File: human-spaceflight/data/MORGAN_AR.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  - name: Soyuz MS-12
+    time_start: 2019-03-14T19:14Z
+    time_end:
+evas:
```
Comment (christiansandberg2045) [inline human-spaceflight/data/MORGAN_AR.md:22]: Also (http://www.spacefacts.de/iss/english/exp_60.htm)
- time_start: 2019-08-21T12:27Z
    time_end:   2019-08-21T18:59Z

File: human-spaceflight/data/PARMITANO_LS.md [hunk 6]
```diff
@@ -17,10 +17,21 @@ missions:
   - name: Soyuz TMA-09M
     time_start:   2013-05-28T20:31:24Z
     time_end:     2013-11-11T02:50Z
+  - name: Soyuz MS-13
+    time_start:   2019-07-20T16:28:21Z
+    time_end:     2020-02-06T00:00Z
```
Comment (christiansandberg2045) [inline human-spaceflight/data/PARMITANO_LS.md]: Should be: 2020-02-06T09:12Z

File: human-spaceflight/data/SKRIPOCHKA_OI.md [hunk 7]
```diff
@@ -19,6 +19,9 @@ missions:
   - name: Soyuz TMA-20M
     time_start: 2016-03-18T21:26Z
     time_end:   2016-09-07T01:13Z
+  - name: Soyuz MS-15
+    time_start: 2019-09-25T13:57:43Z
+    time_end:
```
Comment (christiansandberg2045) [inline human-spaceflight/data/SKRIPOCHKA_OI.md]: time_end: 2020-04-17T05:16Z

File: human-spaceflight/data/SKVORTSOV_AAJ.md [hunk 8]
```diff
@@ -19,6 +19,9 @@ missions:
   - name: Soyuz TMA-12M
     time_start:   2014-03-25T21:17:23Z
     time_end:     2014-09-11T02:23Z
+  - name: Soyuz MS-13
+    time_start:   2019-07-20T16:28:21Z
+    time_end:     2020-02-06T00:00Z
```
Comment (christiansandberg2045) [inline human-spaceflight/data/SKVORTSOV_AAJ.md]: Should be: time_end: 2020-02-06T09:12Z

File: human-spaceflight/data/MORGAN_AR.md (hunk 4 above)
Comment (chrisnorth) [reply human-spaceflight/data/MORGAN_AR.md:15]: oops - copy past error! thanks!

**Comment by chrisnorth** [comment]
@christiansandberg2045 - thanks! Those should all be fixed now.

**Pull request merged**